packages/near-fast-auth-signer/src/components/SignMultichain/bitcoin/schema.ts

```diff
@@ -3,15 +3,32 @@ import * as yup from 'yup';
 import { BaseSendMultichainMessageSchema } from '../schema';
 
 export const SendBTCMultichainMessageSchema = BaseSendMultichainMessageSchema.shape({
-  chain:   yup.number().required().test('is-btc', 'Invalid BTC chain value', (value) => value === 0),
-  network: yup.string().oneOf(['mainnet', 'testnet']).required(),
-  fee:     yup.number().optional(),
-  utxos:   yup.array().optional(),
-}).test('fee and utxos check', 'If fee is present, utxos must be present and vice versa', (value) => {
-  const { fee, utxos } = value;
-  // Check if both are present or both are undefined
-  if ((fee && utxos) || (!fee && !utxos)) {
-    return true; // Validation succeeds
+  transaction: yup.object().shape({
```

Comment:
We should validate that a `from` is provided here. As discussed the dapp can choose to be explicit about the from and if it is provided the signer should check that it matches the derivation path

Reply:
The BTC doesn't accept a from, even if provided it will be ignored, now and before. 

The EVM can accept a from and as discussed it will be validated on the multichain-tools, here is the place where the validation occurs: https://github.com/near/multichain-tools/blob/cce661c13a53b8b89dde5f3c82f3aadd1790a0aa/src/chains/EVM/EVM.ts#L156